index.html

```diff
@@ -1,47 +1,199 @@
+        <p>
+          Student Robotics primes participants for good careers in engineering.
+          Participants assemble a team, build a robot, and compete. All the while
+          gaining valuable technical and soft skills to help their future careers.
+        </p>
+      </div>
+      <div class="row">
+        <div class="column center-label s-4-12">
+          <img src="{{ '/images/advertising/happy-team.jpg' | prepend: site.baseurl }}"
+            alt="A team of students looking happy standing behind their robot" />
+          <h3>Working as a team</h3>
+          <p>
+            For a team to succeed at Student Robotics, they need to work
+            together. For many participants, this is the first time they have worked as part
+            of a team. This practical experience is invaluable for their future careers.
+          </p>
+        </div>
+        <div class="column center-label s-4-12">
+          <img src="{{ '/images/advertising/team-coding.jpg' | prepend: site.baseurl }}"
+            alt="Three competitors sitting in front of a laptop" />
+          <h3>Writing real code</h3>
+          <p>
+            Without some serious code, even the most masterfully built robots
+            can't do very much. With the help of our kit, teams will program
+            their robot in Python to compete in the competition.
+          </p>
+        </div>
+        <div class="column center-label s-4-12">
+          <img src="{{ '/images/advertising/team-building-robot.jpg' | prepend: site.baseurl }}"
+            alt="Students and a teacher sitting around their robot" />
+          <h3>Building a robot</h3>
+          <p>
+            There's nothing quite like building something and bringing it to
+            life. Using our kit, teams will build a fully autonomous robot. The
+            best teams pour their imagination into the design and the look of
+            their robots.
+          </p>
+        </div>
       </div>
     </div>
   </div>
-  <div class="fixed-width">
-    <div class="row">
-      <div class="sidebar column m-4-12">
-        <div class="card">
-          <h3 class="card-heading">Upcoming Events</h3>
+</div>
+<div class="panel">
+  <div class="panel-container">
+    <div class="fixed-width">
+      <h2 class="text-center">Testimonials</h2>
+      <div class="row vertically-centered-row">
+        <div class="column m-9-12">
+          <blockquote class="text-italic">
+            Student Robotics provided me with the invaluable experience of working on a engineering project with a team.
+            I learned not only good engineering principals but the importance of being able to clearly communicate my
+            thoughts and reasoning.
+            Competing in Student Robotics opened the door to me directly entering the world of work as a Software
+            Engineer after completing Sixth Form.
+          </blockquote>
+          <p class="text-heavy text-center">— James Seden Smith</p>
```

Comment:
See my comment regarding names and consistency on the image name. Same applies here

Reply:
Antoine has done this is in 5d2c6fa